core/src/main/java/org/apache/carbondata/core/scan/filter/FilterExpressionProcessor.java

```diff
@@ -163,17 +163,17 @@ public FilterResolverIntf getFilterResolver(Expression expressionTree,
   }
 
   private String joinByteArray(byte[] bytes, char delimiter) {
-    StringBuffer byteArrayAsString = new StringBuffer();
-    byteArrayAsString.append("");
+    String byteArrayAsString = "";
```

Comment:
do not change this to String,Findbugs will give error.
though java optimises internally when strings are concatenated, findbugs doesn't recognise that